Sources/PackageDescription4/Package.swift

```diff
@@ -240,7 +276,19 @@ public final class Package {
         registerExitHandler()
     }
 
-    /// Construct a package.
+    /// Initializes and returns a newly allocated package object with the specified parameters
+    ///
+    /// - Parameters:
+    ///     - name: The name of the Swift package.
+    ///     - platforms: The list of minimum deployment targets per platform.
+    ///     - pkgConfig: Additional flags for a system package.
+    ///     - providers: The package providers for a system package.
```

Comment:
Let's remove doc comments for pkgConfig and providers from initializer for 4.2 and 5 since they are deprecated.